website/docs/tutorials/guides/deployment.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: Deployment
+---
+
+# Deployment
+
+Protostar `>=0.2.1` supports deploying smart contracts to a given network with the `protostar deploy` command. It has similar interface to `starknet deploy` command. [Read the CLI reference for the deploy command](/docs/cli-reference#deploy) to learn more about all supported arguments.
+
+## Example — deploying the default contract
+1. Create new project 
+   ```bash
+   $ protostar init
+   ```
+2. Build the project
+   ```bash
+   $ protostar build
+   ```
```

Comment:
Should we include it in this section or linik init tutorial? Im not sure, your call